src/foam/dao/EasyDAO.js

```diff
@@ -61,22 +61,28 @@ foam.CLASS({
     'foam.dao.RequestResponseClientDAO',
     'foam.dao.SequenceNumberDAO',
     'foam.dao.SyncDAO',
-    'foam.dao.TimingDAO'
+    'foam.dao.TimingDAO',
+    {
+      path: 'foam.dao.JournalType',
+      flags: ['java']
+    },
+    'foam.dao.RoutingJournal',
+    'foam.dao.RoutingJDAO'
   ],
 
-  imports: [ 'document' ],
+  imports: ['document'],
 
   constants: [
     {
       // Aliases for daoType
       name: 'aliases',
-      flags: [ 'js' ],
+      flags: ['js'],
       value: {
-        ARRAY:  'foam.dao.ArrayDAO',
```

Comment:
Personally I would prefer not to mix formatting cleanup PRs with functional changing PRs.  It makes it harder to review.

Reply:
It was my JS Linter. I will bear this in mind for the future.